### ISSUE TITLE
Keymanager key import with slashing protection data fails

### ISSUE THREAD

**Issue by yorickdowne** [status-im/nim-beacon-chain#3984]
**Describe the bug**

When calling keymanager import with data like follows, import fails with error `Invalid keystore objects found`

```
{
  "keystores": [
    "{\"crypto\": {\"kdf\": {\"function\": \"scrypt\", \"params\": {\"dklen\": 32, \"n\": 262144, \"r\": 8, \"p\": 1, \"salt\": \"52eea9787791ad23abb3432c8b9896a58ce77d7762cbef1fde5141aef9df45ec\"}, \"message\": \"\"}, \"checksum\": {\"function\": \"sha256\", \"params\": {}, \"message\": \"59b849d3db1909a83487e25d9da04f9fafabf4fb76f6bd019e62a1fd3b9b4ba3\"}, \"cipher\": {\"function\": \"aes-128-ctr\", \"params\": {\"iv\": \"e0ba50fe6fdfef78c2642c4c4c9cbe9a\"}, \"message\": \"10b357a5cb77097e47fa1b5cd48c7328d104921a5c1f1053a8e1c6d341709c2b\"}}, \"description\": \"\", \"pubkey\": \"9449c954822a6b836fdf1be20a27c3d4ff88d887e3619e998818ffa534b15a1a410d367da0d4ded4984ac747bfb074e4\", \"path\": \"m/12381/3600/2/0/0\", \"uuid\": \"7ab7c5e1-e983-47db-a3c4-df18f5ac649c\", \"version\": 4}"
  ],
  "passwords": [
    "SoVerySecret"
  ],
  "slashing_protection": "{\"metadata\":{\"interchange_format_version\":\"5\",\"genesis_validators_root\":\"0x043db0d9a83813551ee2f33450d23797757d430911a9320530ad8a0eabc43efb\"},\"data\":[{\"pubkey\":\"0x9449c954822a6b836fdf1be20a27c3d4ff88d887e3619e998818ffa534b15a1a410d367da0d4ded4984ac747bfb074e4\",\"signed_blocks\":[{\"slot\":\"0\"}],\"signed_attestations\":[{\"source_epoch\":\"112955\",\"target_epoch\":\"112956\"}]}]}"
}
```

When importing without `slashing_protection`, import is successful.

**Comment by yorickdowne** [comment]
This issue continues with 22.8.0

**Comment by yorickdowne** [comment]
Solved in 22.9.0. Thanks
